dev-libs/benchmark/benchmark-1.3.0.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A microbenchmark support library"
+DESCRIPTION="A library to support the benchmarking of functions, similar to unit-tests."
+HOMEPAGE="https://github.com/google/benchmark"
+COPYRIGHT="2015-2017 Google Benchmark developers"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/google/benchmark/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="f19559475a592cbd5ac48b61f6b9cedf87f0b6775d1443de54cfe8f53940b28d"
```

Comment:
The archive should be renamed to `<port name>-<port version>` as required by the [Guidelines][0]

```diff
+SOURCE_FILENAME="benchmark-$portVersion.tar.gz
```

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name